src/Service.Host/client/src/components/stockLocatorUtility/StockViewerOptions.js

```diff
@@ -53,6 +53,10 @@ function StockViewerOptions({
         }))
     };
 
+    useEffect(() => {
+        document.title = 'Stock Locator Options';
```

Comment:
Mark asked if a few of his most used stores pages could have page titles so that he can easily see which is which when has lots of tabs open, which I agreed to do if it was easy enough.

I can't think of a neater way to do it than in a useEffect... I could just do it in the body of the functional components.. but that would do it every render as opposed to just once, 'componentDidMount' style, which the useEffect with empty dependency array achieves.... Is there an obvious better way I'm missing?

On a related nore, I was thinking about adding a 'title' prop to our common 'Page' component to make it a little simpler in the future.